lib/task-data/schemas/dell-wsman-getInventory.json

```diff
@@ -0,0 +1,29 @@
+{
+    "copyright": "Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.",
+    "definitions": {
+        "verifySSL": {
+            "description": "Specify if SSL certificate is enabled",
+            "type": "boolean"
+        },
+        "_taskTimeout": {
```

Comment:
This is an options for all tasks and it has been covered in common-task-options (https://github.com/RackHD/on-tasks/blob/master/lib/task-data/schemas/common-task-options.json#L22), so you don't need to describe it again.

Reply:
I will remove "_taskTimeout"